{
    "PackageVulnerabilityID": "1",
    "DisclosureDate": "2021-01-26",
    "AffectedVersionRange": "*",
    "Summary": "Arbitrary Code Execution",
    "Contributor": {
        "Discloser": "arjunshibu",
        "Fixer": ""
    },
    "Package": {
        "Registry": "pip",
        "Name": "tensorflow",
        "URL": "https://pypi.org/project/tensorflow",
        "Downloads": ""
    },
    "CWEs": [
        {
            "ID": "94",
            "Description": "Arbitrary Code Execution"
        }
    ],
    "CVSS": {
        "Version": "3.1",
        "AV": "N",
        "AC": "L",
        "PR": "N",
        "UI": "R",
        "S": "U",
        "C": "L",
        "I": "L",
        "A": "N",
        "E": "",
        "RL": "",
        "RC": "",
        "Score": "8.1"
    },
    "CVEs": [
        ""
    ],
    "Repository": {
        "URL": "https://github.com/tensorflow/tensorflow",
        "Codebase": [
            "C++", "Python"
        ],
        "Owner": "tensorflow",
        "Name": "tensorflow",
        "Forks": "83800",
        "Stars": "153000"
    },
    "Permalinks": [
        "https://user-images.githubusercontent.com/43996156/105678852-a019e100-5f13-11eb-992c-a998d626f2a5.png"
    ],
    "References": [],
    "PrNumber": ""
}
